demo/kserve/scripts/install/kserve-install.sh

```diff
@@ -14,9 +14,9 @@ then
   input="y"
 else
   echo "** Check User Workload Configmap for Kserve metrics before you execute this script **"
-  echo 
+  echo
+  # TODO point of improvement, execute the command and bring the real output from user's cluster.
```

Comment:
Is this required?

Reply:
No, this was edited by Jooho in his last PR.